Userland/Applications/Browser/Tab.cpp

```diff
@@ -407,7 +407,10 @@ Optional<URL> Tab::url_from_location_bar(MayAppendTLD may_append_tld)
     String text = m_location_box->text();
 
     StringBuilder builder;
-    builder.append(text);
+
+    String text_url = g_search_engine.replace("{}", text);
+    builder.append(text_url);
```

Comment:
This looks like it will just wrap everything into a search query, even if it's a proper URL.